apps/explorer/src/core/components/props/index.ts

```diff
@@ -19,7 +19,7 @@ export interface NavHeader {
 
 export interface Footnote {
   color?: string
-  text?: string
+  text?: TranslateResult
```

Comment:
Be careful with this change as you will affect other components using Footnote (and you're changing only one).

Said that, you can modify Footnote to use the following:

```
export interface Footnote {
  text?: TranslateResult | string
}
```

So it accept both formats and the other components are not affected by this change.

Or the other possibility is to take a look on other components that are using Footnote and change them.

Reply:
I've pushed the change to the other components instead of using toString() method (take a look on my commit)

Reply:
I will take a look at it :)